socorro/signature/__main__.py

```diff
@@ -31,7 +31,7 @@
 
 
 # FIXME(willkg): This hits production. We might want it configurable.
-API_URL = 'https://crash-stats.mozilla.com/api/'
+API_URL = 'https://crash-stats.mozilla.com/api'
```

Comment:
The `/` at the end of this caused the urls to be like `https://crash-stats.mozilla.com/api//RawCrash/` which raises a 404 now. I don't know why it didn't do that before or when it changed. Anyhow, yay.